docs/en/observability/monitor-azure-agent.asciidoc

```diff
@@ -0,0 +1,382 @@
+[[monitor-azure-elastic-agent]]
+== Monitor Microsoft Azure with {agent}
+
+****
+**New to Elastic?** Follow the steps in our {estc-welcome}/getting-started-observability.html[getting started guide] instead
+of the steps described here. Return to this tutorial after you've learned the
+basics.
+
+**Using the native Azure integration from the marketplace?** Refer to
+<<monitor-azure-native>>.
+****
+
+In this tutorial, you’ll learn how to deploy {agent} and monitor your Azure
+infrastructure with Elastic {observability}.
+
+[discrete]
+[[azure-elastic-agent-what-you-learn]]
+=== What you'll learn
+
+You'll learn how to:
+
+* Create an Azure service principal with permissions to read monitoring data.
+* Collect Azure billing metrics.
+* Collect Azure activity logs.
+* Visualize the logs and infrastructure metrics in {kib}.
+
+[discrete]
+[[azure-collect-metrics]]
+=== Step 1: Create an Azure service principal
+
+In this step, you create an Azure service principal and then grant them access
+to use the Azure Monitor REST API.
+
+The https://docs.microsoft.com/en-us/rest/api/monitor/[Azure Monitor REST API]
+allows you to get insights into your Azure resources using different operations.
+To access the Azure Monitor REST API, you need to use the Azure Resource Manager
+authentication model. Therefore, you must authenticate all requests with Azure
```

Comment:
@dedemorton, in this section, you mention "Azure Monitor REST API." 

I believe we can use the more general name "Azure REST API." The [Azure REST API](https://learn.microsoft.com/en-us/rest/api/azure/) is not a specific API endpoint, but the collection of all actual endpoints and [Azure Monitor REST API](https://learn.microsoft.com/en-us/rest/api/monitor/) is one of them.  

For example, the Azure Billing Metrics uses two different Azure APIs: Consumption API and Cost Management API.

Depending on their permission, app registrations can access a wide range of APIs.